substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/traceid/JfrTraceIdEpoch.java

```diff
@@ -43,6 +43,7 @@ public class JfrTraceIdEpoch {
     private static final long EPOCH_1_BIT = 0b10;
 
     private boolean epoch;
```

Comment:
`epoch` can probably be removed (i.e., can be computed from the least significant bit of `epochGeneration`).

Reply:
Ok, good idea. I'll remove that.